docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
 
 We (Student Robotics) manage the charging of all batteries at the competition. Battery Charging volunteers are responsible for ensuring that there is a continuous supply of charged batteries available for use by teams.
 
-## Equipment
+## Setup
 
- * Battery chargers
- * Discharged battery box
- * Charged battery box
- * Damaged battery/charger box
- * Red insulating tape
- * Example good batteries (one brand new, one squidgy)
```

Comment:
Why are we removing these example good batteries from the requirements?

Reply:
This was an oversight on my part when making these changes. I will add them to the equipment for charging station.